### PR TITLE
Feature/bye travis the flash

### PR DESCRIPTION
**Pull request by chenxsan**
Time to get rid of travis from `master` branch as well.

### CODE REVIEW

**Comment by jeffin143** [comment]
Nice name for the branch :)

File: package.json [hunk 1]
```diff
@@ -51,7 +51,6 @@
     "lint:links": "hyperlink -c 8 --root dist -r dist/index.html --canonicalroot https://webpack.js.org/ --internal --skip /plugins/extract-text-webpack-plugin/ --skip /printable --skip https:// --skip http:// --skip sw.js > internal-links.tap; cat internal-links.tap | tap-spot",
     "sitemap": "cd dist && sitemap-static --ignore-file=../sitemap-ignore.json --pretty --prefix=https://webpack.js.org/ > sitemap.xml",
     "serve": "npm run build && sirv start ./dist --port 4000",
-    "deploy": "gh-pages -d dist",
```
Comment (montogeek) [inline package.json:54]: Uhmm, What if you need to deploy/force it?

Comment (chenxsan) [reply package.json:54]: In which case do you need to deploy/force it? It's a bad idea to deploy without running against CI. If you must, feel free to add it back.

Comment (montogeek) [reply package.json:54]: In the case the CI is not available.

**Pull request merged**